css/style.css

```diff
@@ -941,30 +926,52 @@ details[open] summary::after {
   grid-column: span 2;
 }
 
-
 .form-grid {
   display: grid;
-  grid-template-columns: 1fr 2fr;
-  grid-template-rows: auto auto auto;
-  
-  padding: 0;
-  margin: 0;
-  gap: 20px;
+  grid-template-columns: 1fr 1fr;
+  grid-gap: 20px;
+  justify-items: stretch;
 }
+
 .message-item {
-  grid-column: span 1;
+  display: block;
 }
 
-.message-email {
+.message-item input, 
+.message-message {
+  width: 100%;
+  padding: 14px 24px;
+  box-sizing: border-box;
 
+  color: #C2C3C5; 
+  background-color: #28282C;
+  border: none;
 }
 
- .message-phone, .message-message, .message-button  {
-  grid-column: span 2;
+.message-item input::placeholder,
+.message-message::placeholder {
+  color: #C2C3C5; 
+  background-color: #28282C;
+}
+
+.message-name {
+  grid-column: 1 / 2;
+}
+
+.message-email {
+  grid-column: 2 / 3; 
+}
+
+.message-phone {
+  grid-column: 1 / 3;
```

Comment:
You're applying grid styles to inputs, but they're inside of labels, so grid properties doesn't work.

So, now phone input taking only half of grid container width.

And another thing that I missed before: your textarea is without label. Could you please add it?